### PR TITLE
Add a setting to globally disable quiet subscriptions

### PR DESCRIPTION
**Pull request by ldidry**

### CODE REVIEW

File: src/lib/Sympa/ConfDef.pm [hunk 1]
```diff
@@ -2132,6 +2132,14 @@ our @params = (
         'file'     => 'sympa.conf',
         'optional' => 1,
     },
+    {   'name'       => 'disable_quiet_subscriptions',
+        'gettext_id' => 'Disable quiet subscriptions feature',
+        'gettext_comment' =>
+            'All subscribers will be notified at subscription time, enhance GPDR compliance',
```
[the remark this answers is not in the document]
Comment (ldidry) [reply src/lib/Sympa/ConfDef.pm]: Thank for noticing. As I'm french, I always speak about the RGPD (the french acronym) and didn't knew exactly what the english acronym is.

File: src/lib/Sympa/Request/Handler/add.pm [hunk 2]
```diff
@@ -180,7 +181,7 @@ Sympa::Request::Handler::add - add request handler
 
 Adds a user to a list (requested by another user). Verifies
 the proper authorization and sends acknowledgements unless
-quiet add.
+quiet add (and disable_quiet_subscriptions setting is false).
```
[the remark this answers is not in the document]
Comment (ldidry) [reply src/lib/Sympa/Request/Handler/add.pm]: :+1:

**Comment by ikedas** [comment]
It's nice to prevent silent addition.  However, in some cases (e.g. with a list system dedicated to one enterprise), disabling _verbose_ subscription is desirable.  My idea is introducing an option `quiet_subscription off|on|optional` (`optional` by default).

**Comment by ldidry** [comment]
Give me a few days, I'll do it friday.

**Comment by ldidry** [comment]
I opened an issue. Closing this PR, will open a new one once it's done.

**Pull request closed without merging**